boards/tsi/ek_tsi_skyp/ek_tsi_skyp.yaml

```diff
@@ -0,0 +1,19 @@
+identifier: ek_tsi_skyp
+name: TSI sky-p
+type: mcu
+arch: arm
+ram: 2016
+flash: 32
```

Comment:
So here we have reworked the RAM and FLASH space check the entries in sram0 and sram and we have kept it as 256 K Flash and SRAM i.e. RAM as 1792.